virttest/libvirt_xml/vm_xml.py

```diff
@@ -2091,6 +2091,18 @@ def remove_numa_cells(self):
             pass  # Element already doesn't exist
         self.xmltreefile.write()
 
+    def remove_elem_by_xpath(self, xpath_to_remove):
+        """
+        Remove a specified sub element from cpu configuration
+
+        :param xpath_to_remove:  str, like '/model' '/numa'
+        """
+        try:
+            self.xmltreefile.remove_by_xpath(xpath_to_remove, remove_all=True)
```

Comment:
I think it would be good to have remove_all parameter configurable for this method.

Reply:
Good catch. I enhanced it. Thanks.